version.txt

```diff
@@ -1 +1 @@
-21.0.4.0.4+7-GA
\ No newline at end of file
+21.0.5.0.5+9-GA
```

Comment:
diff --git a/version.txt b/version.txt
index dec00a8e732..78a582d5ce8 100644
--- a/version.txt
+++ b/version.txt
@@ -1 +1 @@
-21.0.5.0.5+9-GA
\ No newline at end of file
+21.0.4.0.4+7-GA
\ No newline at end of file